pkg/commands/git_commands/branch_loader_test.go

```diff
@@ -18,7 +18,7 @@ func TestObtainBranch(t *testing.T) {
 	scenarios := []scenario{
 		{
 			testName: "TrimHeads",
-			input:    []string{"", "heads/a_branch", "", "", "subject", "123"},
+			input:    []string{"", "heads/a_branch", "", "", "subject", "123", ""},
```

Comment:
We are mocking what git returns to us here, and an empty string is very unrealistic, git will never return that. I'd suggest to return a real number that could happen in practice; one way to do that would be
```
	now := time.Now().Unix()
	twoAndAHalfHoursAgo := strconv.Itoa(int(now - 2.5*60*60))
```
This will result in a recency string of `2h`.

If you follow the suggestion to add a `storeCommitterDateAsRecency` parameter, it would be good to add that to the scenario struct and add one more test case where it is true.

Reply:
Fixed in https://github.com/jesseduffield/lazygit/commit/bea96a5babac6ae2a941e99a01446d7c23fc2c8e#diff-db04dda60e2b65b04e8136c601b4549909b8f0bd6c7dbc852fb436374b24ab6fR91